docs/base_models/seggpt.md

```diff
@@ -0,0 +1,64 @@
+# SegGPT
+
+[SegGPT](https://github.com/baaivision/Painter/tree/main/SegGPT) is a transformer-based few-shot semantic segmentation model developed by [BAAI Vision](https://github.com/baaivision).
+
+This model performs well on task-specific segmentation tasks when given a few labeled images from which to learn features about the objects you want to identify.
+
+## Installation
+
+To use SegGPT with autodistill, you need to install the following dependency:
+
+
+```bash
+pip3 install autodistill-seggpt
+```
+
+## About SegGPT
+
+This Autodistill module uses a handful of pre-labelled images for improved accuracy.
```

Comment:
Does this module require images for improved accuracy?

Reply:
How do you feel about the new phrasing?

> SegGPT performs "in-context" segmentation. This means it requires a handful of pre-labelled "context" images.